player/lua/stats.lua

```diff
@@ -1721,20 +1721,20 @@ mp.add_key_binding(nil, "display-stats", function() process_key_binding(true) en
 mp.add_key_binding(nil, "display-stats-toggle", function() process_key_binding(false) end,
     {repeatable=false})
 
-for k, _ in pairs(pages) do
+for i = 0, 5 do
```

Comment:
this should take the length form `pages` instead magic number.

Reply:
`#pages` => `0`, how to take the length in lua?